drivers/sensor/ism330dhcx/ism330dhcx.c

```diff
@@ -748,11 +748,9 @@ static const struct ism330dhcx_config ism330dhcx_config = {
 #else
 #error "BUS MACRO NOT DEFINED IN DTS"
 #endif
+	IF_ENABLED(CONFIG_ISM330DHCX_TRIGGER,
+		   (.drdy_gpio = GPIO_DT_SPEC_INST_GET(0, drdy_gpios),))
 #ifdef CONFIG_ISM330DHCX_TRIGGER
-	/* One gpio pin declared in DTS */
-	.int_gpio_port = DT_INST_GPIO_LABEL(0, drdy_gpios),
-	.int_gpio_pin = DT_INST_GPIO_PIN(0, drdy_gpios),
-	.int_gpio_flags = DT_INST_GPIO_FLAGS(0, drdy_gpios),
 	.int_pin = DT_INST_PROP(0, int_pin),
```

Comment:
you should embed this into the previous `IF_ENABLED` as well

Reply:
```c
IF_ENABLED(CONFIG_ISM330DHCX_TRIGGER,
                       (.drdy_gpios = ...,
                         .int_pin = ...,))
```

Reply:
Fixed!